src/main/java/org/retest/rebazer/service/RebaseService.java

```diff
@@ -147,7 +147,8 @@ public boolean rebase( final RepositoryConfig repoConfig, final PullRequest pull
 
 				case FAST_FORWARD:
 					log.warn( "Why creating {} without changes?", pullRequest );
-					// fall-through
+					localRepo.push().setCredentialsProvider( credential ).setForce( true ).call();
+					return true;
```

Comment:
Why not fall through? Is this better readable?

Reply:
Prevent compiler warnings in restrictive setup, `//$FALL-THROUGH$` is supported by eclipse compiler but not by javac